assets/js/base/components/cart-checkout/product-name/index.js

```diff
@@ -9,8 +9,13 @@ import { decodeEntities } from '@wordpress/html-entities';
  */
 import './style.scss';
 
-const ProductName = ( { name, permalink, disabled = false } ) => {
-	return (
+const ProductName = ( {
+	name,
+	permalink,
+	disabled = false,
+	hasLink = true,
+} ) => {
+	return hasLink ? (
 		// we use tabIndex -1 to prevent the link from being focused, pointer-events
 		// disabled click events, so we get an almost disabled link.
```

Comment:
Would using `disabled` not be enough?

Reply:
It seems `disabled` doesn't stop people clicking on the link, it stops people _tabbing_ to the link but not clicking it. I notice the comment 

```
// we use tabIndex -1 to prevent the link from being focused, pointer-events
// disabled click events, so we get an almost disabled link.
```

but notice also that `pointer-events: none` never gets applied to the `a` here. 

My counter question is: why do we keep the `a` here if it's supposed to be disabled?

I propose that we use the `disabled` prop, so no need to add `hasLink` but keep the logic that outputs a `span` if the link is disabled, or outputs an `a` if it is not.